.github/workflows/docker.yml

```diff
@@ -0,0 +1,64 @@
+name: Docker Image
+# Note: this is taken from Gaia's docker image push, it has nice support for tagged releases.
+
+on:
+  pull_request:
+  push:
+    branches:
+      - '*'  
+    tags:
+      - "v[0-9]+.[0-9]+.[0-9]+" # Push events to matching v*, i.e. v1.0, v20.15.10
+      - "v[0-9]+.[0-9]+.[0-9]+-rc*" # Push events to matching v*, i.e. v1.0-rc1, v20.15.10-rc5
+
+jobs:
+  docker:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@master
+        with:
+          fetch-depth: 0
+
+      - name: Prepare
+        id: prep
+        run: |
+          DOCKER_IMAGE=faddat/wasmd
```

Comment:
We publish our "official" wasmd image to https://hub.docker.com/r/cosmwasm/wasmd/tags?page=1&ordering=last_updated